<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MeltAway</class>
 <widget class="QMainWindow" name="MeltAway">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MeltAway</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>6</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetMinimumSize</enum>
      </property>
      <item>
       <widget class="QLineEdit" name="lineEdit"/>
      </item>
      <item>
       <widget class="QPushButton" name="OpenButton">
        <property name="text">
         <string>Открыть</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="WidgetGL3D" name="openGLWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="topMargin">
       <number>10</number>
      </property>
      <property name="bottomMargin">
       <number>10</number>
      </property>
      <item>
       <widget class="QSlider" name="horizontalSlider">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox_BrushSelection">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Brush Selection</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_file">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="separator"/>
    <addaction name="action_exit"/>
   </widget>
   <widget class="QMenu" name="menu_view">
    <property name="title">
     <string>Вид</string>
    </property>
    <widget class="QMenu" name="menu_viewType">
     <property name="title">
      <string>Режим отображения</string>
     </property>
     <addaction name="action_setViewType_solid"/>
     <addaction name="action_setViewType_wire"/>
    </widget>
    <addaction name="menu_viewType"/>
   </widget>
   <addaction name="menu_file"/>
   <addaction name="menu_view"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_exit">
   <property name="text">
    <string>Выйти</string>
   </property>
   <property name="toolTip">
    <string>Выйти из приложения</string>
   </property>
  </action>
  <action name="action_setViewType_solid">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Твердотельный</string>
   </property>
  </action>
  <action name="action_setViewType_wire">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Сеточный</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>WidgetGL3D</class>
   <extends>QOpenGLWidget</extends>
   <header>widgetgl3d.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
